From 375c0279b6afa59ea76b0937a243229c4dad1861 Mon Sep 17 00:00:00 2001
From: Timo Bruderek <openvario@timoshome.org>
Date: Mon, 14 Sep 2026 17:12:04 +0000
Subject: [PATCH 2/2] Environment-Openvario-mainline

Rebased onto u-boot v2024.01; the surrounding code moved but the change
itself is unaltered from the version that applied to 2022.01.
---
 include/configs/sunxi-common.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/sunxi-common.h b/include/configs/sunxi-common.h
index b8ca77d0..76f3aa2b 100644
--- a/include/configs/sunxi-common.h
+++ b/include/configs/sunxi-common.h
@@ -286,8 +286,8 @@
 
 #ifdef CONFIG_VIDEO
 #define CONSOLE_STDOUT_SETTINGS \
-	"stdout=serial,vidconsole\0" \
-	"stderr=serial,vidconsole\0"
+	"stdout=serial\0" \
+	"stderr=serial\0"
 #else
 #define CONSOLE_STDOUT_SETTINGS \
 	"stdout=serial\0" \
